### DISCUSSION TITLE
Regarding the size of the pre loaded maps

### DISCUSSION THREAD

**Discussion by AbhayGoyal** [akeranen/the-one#116] [General]
Hi,
there are multiple pre-loaded maps available with the one simulator. Does anyone know the exact dimensions of these things in like sq miles or how they actually measure in the real world?

**Comment by akeranen** [comment]
The coordinates in the maps are in meters and usually translated to start at (0,0) so you can look up the largest X and Y values in the map file and calculate the dimension accordingly.
